### PR TITLE
Potential solution for #134

### PR DESCRIPTION
**Pull request by Lazarix**
Potential solution for #134 
If i'm missing other changes because of moving tritanium_ingot.json to tritanium_ingot_nugget.json, let me know

### CODE REVIEW

**Comment by adrwa** [comment]
Have you actually checked that this works because looking at your "fix" i can see there's a 100% chance that it does not work

File: src/main/resources/assets/matteroverdrive/recipes/tritanium_ingot.json [hunk 1]
```diff
@@ -2,15 +2,12 @@
   "result": {
     "item": "matteroverdrive:tritanium_ingot"
```
Comment (ChloeDawn) [inline src/main/resources/assets/matteroverdrive/recipes/tritanium_ingot.json:3]: This recipe only outputs 1 ingot.

File: src/main/resources/assets/matteroverdrive/recipes/tritanium_ingot.json [hunk 2]
```diff
@@ -2,15 +2,12 @@
...
-  "type": "minecraft:crafting_shaped",
-  "key": {
-    "#": {
-      "item": "#NUGGETTRITANIUM"
-    }
+  "type": "forge:ore_shapeless",
+  "ingredients": [
+        {
+            "type": "forge:ore_dict",
+            "ore": "matteroverdrive:tritanium_block"
```
Comment (ChloeDawn) [inline src/main/resources/assets/matteroverdrive/recipes/tritanium_ingot.json:9]: This is not a valid ore name.

**Review by ChloeDawn — CHANGES_REQUESTED** [review]

**Comment by Lazarix** [comment]
well, an effort was made.

**Pull request closed without merging**